features/admin.users.v1/components/user-profile.tsx

```diff
@@ -525,18 +525,42 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
      *
      * @param locale - locale value.
      * @param localeJoiningSymbol - symbol used to join language and region parts of locale.
+     * @param updateSupportedLanguage - If supported languages needs to be updated with the given localString or not.
      */
-    const  normalizeLocaleFormat = (locale: string, localeJoiningSymbol: LocaleJoiningSymbol): string => {
-        if (!locale) {
-            return locale;
+    const normalizeLocaleFormat = (
+        localeString: string,
+        localeJoiningSymbol: LocaleJoiningSymbol,
+        updateSupportedLanguage: boolean
+    ): string => {
+        if (!localeString) {
+            return localeString;
         }
 
-        let [ language, region ] = locale.split(/[-_]/);
+        const separatorIndex: number = localeString.search(/[-_]/);
 
-        language = language.toLowerCase();
-        region = region.toUpperCase();
+        let language: string;
+        let region: string;
+        let normalizedLocale: string;
```

Comment:
```suggestion
        let normalizedLocale: string = localString
```

and remove the else block

Reply:
Fixed